app/helpers/ipp_helper.rb

```diff
@@ -8,8 +8,12 @@ def scrub_message(message)
   def scrub_body(body)
     return nil if body.nil?
 
-    body = body.with_indifferent_access
-    body[:responseMessage] = scrub_message(body[:responseMessage])
-    body
+    if body.is_a?(String)
```

Comment:
[non-blocking] I would be curious when a `String` would come from USPS. I don't remember seeing anything explicit about it in the API docs 🤔

Reply:
I am too. 🤔  That's the only thing lingering in my head for this as well, and I wasn't sure if it warranted more concern. But I wasn't able to replicate it based on the user flows I dug up leading to this error, so I couldn't determine if it was something we aren't anticipating on our end. Either way I figure it can just be scrubbed.

Reply:
@jennyverdeyen If you want, we could email USPS or ask about it at one of our technical sync meetings.  We'd probably get a response sooner if we reach out via email.